components/image/image.css

```diff
@@ -0,0 +1,13 @@
+.responsive-img{
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+}
+
+.round-img{
+    height: 10rem;
+    width: 10rem;
+    border-radius: 50%;
```

Comment:
you can declare a global class `rounded` with border-radius: 50%, it seems it will be used in a lot of places.

Reply:
ok! yeah will do!